src/app/shared/error-toasters/general-message-error/general-message-error.component.html

```diff
@@ -1,2 +1,15 @@
-<p style="display: inline-block; max-width: 300px">{{ data.message }}</p>
-<button style="float: right; margin-top: 8px" mat-button color="accent" (click)="snackBarRef.dismiss()">Close</button>
+<div style="display: flex; align-items: center; justify-content: space-between">
+  <div style="vertical-align: middle; flex-basis: 80%; overflow-wrap: anywhere">
+    <p style="margin: auto 5px; text-align: center">
+      {{ data.message }}
+    </p>
+  </div>
+  <button
+    style="vertical-align: middle; flex-basis: 20%; margin: auto 5px; text-align: center"
+    mat-button
+    color="accent"
+    (click)="snackBarRef.dismiss()"
```

Comment:
Maybe we can abstract this out into a simple method.

Reply:
What do you mean by "simple method"?

Reply:
Oh what I meant is not exposing the inner implementation details. So instead of doing `snackBarRef.dismiss()` we can do something like 

```
(click)="dismissSnackBar()"
```

But I think this is not necessary so I shall resolve the 2 conversations 🤣